specs/Qowaiv.Specs/_Legacy/Mathematics/FractionOperationsTest.cs

```diff
@@ -1,210 +0,0 @@
-﻿namespace Qowaiv.UnitTests.Mathematics;
-
-public class FractionOperationsTest
```

Comment:
Do the operation tests return somewhere?

Reply:
Yep. In the tests you reviewed. :)